### PR TITLE
Common stepper/tabs component

### PR DESCRIPTION
**Pull request by Jood80**
- [x] fill each tab with its relative content.
 Note: I have added some dummy content instead of the real content to check its appearance or if it shows in the tab content fields  
- [x] make the transition between tabs
- [x] including Back and Next buttons

### CODE REVIEW

File: client/src/components/commonComponents/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Paper from '@material-ui/core/Paper';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
```
Comment (sara219) [inline client/src/components/commonComponents/Tabs/index.js]: replace all of that imports with one import from the '@material-ui/core'
and put all of the things you need inside { } 

`import { Toolbar, Step ... } from '@material-ui/core';`

File: client/src/components/commonComponents/Tabs/index.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+          <Typography variant="h6" color="inherit" noWrap>
+            Urban Natural
+          </Typography>
+        </Toolbar>
+      </AppBar>
+
+      <main className={classes.layout}>
+        <Paper className={classes.paper}>
+          <Typography component="h1" variant="h4" align="center">
+            Checkout
```
Comment (sara219) [inline client/src/components/commonComponents/Tabs/index.js]: Can you give it another heading than 'checkout'

**Review by sara219 — CHANGES_REQUESTED** [review]

File: client/src/components/commonComponents/Tabs/index.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import Paper from '@material-ui/core/Paper';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './style';
+
+function Copyright() {
```
Comment (Rahaf-96) [inline client/src/components/commonComponents/Tabs/index.js]: Could you change this function to arrow function please :smiley_cat:

File: client/src/components/commonComponents/Tabs/index.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
+
+const tabs = ['Contact Info', 'Questions', 'Book'];
+
+function getTabContent(tab) {
```
Comment (Rahaf-96) [inline client/src/components/commonComponents/Tabs/index.js]: This one too, change it to arrow function :smile_cat:

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]

File: client/src/components/commonComponents/Tabs/index.js (hunk 3 above)
Comment (Jood80) [reply client/src/components/commonComponents/Tabs/index.js]: Donee :metal:

File: client/src/components/commonComponents/Tabs/index.js (hunk 2 above)
Comment (Jood80) [reply client/src/components/commonComponents/Tabs/index.js]: yes, I put that into consideration too. I tried to change it to "Reservation" but it wasn't convenient with the page. Deleting it was not a very good choice either, so I will think of a better header when we combined it with the NavBar

File: client/src/components/commonComponents/Tabs/index.js (hunk 1 above)
Comment (Jood80) [reply client/src/components/commonComponents/Tabs/index.js]: alright :+1:

**Review by sara219 — APPROVED** [review]

**Review by tasneembhiri — APPROVED** [review]
great job :wink:

**Review by Rahaf-96 — APPROVED** [review]
:clap: :woman_dancing:

**Pull request merged**